.github/workflows/build.yml

```diff
@@ -310,10 +310,9 @@ jobs:
 
             if [ $attempt -lt 10 ]
             then
-              seconds=$((1 + $RANDOM % 30)) # Get random seconds for minor variance
               echo "Broken pipe detected! will try again in $minutes minutes & $seconds seconds."
-              sleeptime=$(($minutes * 60 + seconds)) # Convert to seconds
-              minutes=$((minutes * 2)) # Double minutes in event of another retry
+              sleeptime=$($minutes * 60) # Convert to seconds
+              minutes=$((minutes * 1.5))
```

Comment:
To my knowledge, bash does not support floating point numbers. How about we simply add a minute each time instead?

This would result in a total sleep time of 45 minutes (in the event of 10 retries) with the longest individual wait being 7.5 minutes.

Reply:
e712c85c7621f9fd16773e7666c7944efd94dbe2